packages/toolpad-utils/src/fs.ts

```diff
@@ -3,8 +3,22 @@ import * as path from 'path';
 import { Dirent } from 'fs';
 import * as yaml from 'yaml';
 import { yamlOverwrite } from 'yaml-diff-patch';
+import * as prettier from 'prettier';
 import { errorFrom } from './errors';
 
+/**
+ * Formats a yaml source with `prettier`.
+ */
+const PRETTIERNAME = 'prettier.config.js';
+
+export function formatYaml(code: string): string {
+  const readConfig = prettier.resolveConfig.sync(PRETTIERNAME);
```

Comment:
Upcoming `prettier` update will remove all sync apis. So to make this function forward compatible I'd make `formatYaml` an async function and use the async version of `resolveConfig`. 

In general you can assume that in most of the application code we want to use non-blocking APIs.

Reply:
If I understand correctly, the filePath that is supplied to `resolveConfig` is the one of the file that is to be formatted. The prettier config file is then resolved from that location.

Reply:
It's my first to invoke this function, I try it late